prisma.config.ts

```
@@ -0,0 +1,14 @@
+// This file was generated by Prisma, and assumes you have installed the following:
+// npm install --save-dev prisma dotenv
+import "dotenv/config";
```

ESLint

[error] Replace `"dotenv/config"` with `'dotenv/config'`

(prettier/prettier)